docs/operations/rwx.md

```diff
@@ -1,136 +1,431 @@
+
+ ```bash
+ # Get the pod name of the Ondat CLI utility.
+
+ kubectl get pods --namespace storageos | grep "storageos-cli"
+ storageos-cli-578c4f4674-wr9z2                       1/1     Running   0              3m43s
+ ```
+ 
+ ```bash
+ # Get the volumes in the "ondat-files-squash-mode" namespace using the Ondat CLI.
+ kubectl --namespace=storageos exec storageos-cli-578c4f4674-wr9z2 -- storageos get volumes --namespace=ondat-files-squash-mode
+
+ NAMESPACE                NAME                                      SIZE     LOCATION                                  ATTACHED ON                      REPLICAS  AGE
+ ondat-files-squash-mode  pvc-bd4c93a5-0071-4a56-968c-ae5e12d34342  5.0 GiB  aks-storage-32661963-vmss000001 (online)  aks-storage-32661963-vmss000001  0/0       15 minutes ago
+
+ # Describe the "pvc-bd4c93a5-0071-4a56-968c-ae5e12d34342" volume.
+ kubectl --namespace=storageos exec storageos-cli-578c4f4674-wr9z2 -- storageos describe volume pvc-bd4c93a5-0071-4a56-968c-ae5e12d34342 --namespace=ondat-files-squash-mode
+
+ ID                      17450a4e-ecf6-4e98-918f-19e748bf9737
+ Name                    pvc-bd4c93a5-0071-4a56-968c-ae5e12d34342
+ Description
+ AttachedOn              aks-storage-32661963-vmss000001 (acdf3a74-4042-492d-b7b4-8368f5474fb2)
+ Attachment Type         nfs
+ NFS
+   Service Endpoint      10.224.0.8:25805
+   Exports:
+   - ID                  1
+     Path                /
+     Pseudo Path         /
+     ACLs
+     - Identity Type     hostname
+       Identity Matcher  *
+       Squash            rootuid
+       Squash UID        0
+       Squash GUID       0
+ Namespace               ondat-files-squash-mode (39d924f6-2d16-48e6-af1a-a69169f4bd6b)
+ Labels                  csi.storage.k8s.io/pv/name=pvc-bd4c93a5-0071-4a56-968c-ae5e12d34342,
+                         csi.storage.k8s.io/pvc/name=ondat-files-squash-mode,
+                         csi.storage.k8s.io/pvc/namespace=ondat-files-squash-mode,
+                         storageos.com/nfs-squash=rootuid,
+                         storageos.com/nfs/mount-endpoint=10.0.159.162:2049,
+                         storageos.com/nocompress=true
+ Filesystem              ext4
+ Size                    5.0 GiB (5368709120 bytes)
+ Version                 NQ
+ Created at              2022-09-08T17:07:35Z (17 minutes ago)
+ Updated at              2022-09-08T17:15:36Z (8 minutes ago)
+
+ Master:
+   ID                    662ac10a-b695-4c5f-ba4e-2b003b098dfc
+   Node                  aks-storage-32661963-vmss000001 (acdf3a74-4042-492d-b7b4-8368f5474fb2)
+   Health                online
+ ```
+
+1. If you notice under `Attachment Type         nfs`, detailed information about the RWX volume is returned.
```

Comment:
same question about the number of spaces

Reply:
I copied it directly from the cli output. After rendering, the spaces are removed.